src/routes/notifications/v2/notifications.module.ts

```diff
@@ -1,9 +1,9 @@
+import { Module } from '@nestjs/common';
+import { AuthGuard } from '@/routes/auth/guards/auth.guard';
 import { AuthRepositoryModule } from '@/domain/auth/auth.repository.interface';
+import { NotificationsServiceV2 } from '@/routes/notifications/v2/notifications.service';
+import { NotificationsControllerV2 } from '@/routes/notifications/v2/notifications.controller';
```

Comment:
Are you using an import order organiser?

Reply:
I am the import order organizer 😆

Reply:
I usually do it this way for new files, but I won't change the existing ones. Since I moved this module to a new file, I also changed the import order.